src/types/page.urls.ts

```diff
@@ -218,7 +218,7 @@ export const UPDATE_DIRECTOR_CORRESPONDENCE_ADDRESS_SEARCH_CHOOSE_ADDRESS_PATH_E
 export const UPDATE_DIRECTOR_CORRESPONDENCE_ADDRESS_SEARCH_CHOOSE_ADDRESS = ACTIVE_SUBMISSION_BASE + UPDATE_DIRECTOR_CORRESPONDENCE_ADDRESS_SEARCH_CHOOSE_ADDRESS_PATH_END;
 export const UPDATE_DIRECTOR_CORRESPONDENCE_ADDRESS_SEARCH_CHOOSE_ADDRESS_PATH = OFFICER_FILING + UPDATE_DIRECTOR_CORRESPONDENCE_ADDRESS_SEARCH_CHOOSE_ADDRESS;
 
-export const UPDATE_DIRECTOR_CORRESPONDENCE_ADDRESS_LINK_PATH_END = "link-correspondence-update";
+export const UPDATE_DIRECTOR_CORRESPONDENCE_ADDRESS_LINK_PATH_END = "/link-correspondence-address-update";
```

Comment:
should we have `UPDATE_PATH + link-correspondence-address-update`

Reply:
I queried the address with Matt here
https://companieshouse.atlassian.net/browse/DACT-1700?focusedCommentId=234625